README.md

```diff
@@ -1,20 +1,32 @@
 <h1 align='center'>:cherries: Cherry in PyTorch</h1>
 
-Reinforcement Learning &amp; other bedtime stories in PyTorch. This repository is [pytorch](https://pytorch.org) implementation of the [Intro RL course](https://github.com/simoninithomas/Deep_reinforcement_learning_Course) & [Advanced RL course](https://github.com/dennybritz/reinforcement-learning). Before training/playing the agent please install the external [dependencies.](https://github.com/moabitcoin/cherry-pytorch#dependencies) The name cherry is derived from Yann Lecun's [statement](https://www.quora.com/What-do-you-think-about-reinforcement-learning-Is-it-the-cherry-on-the-cake-as-Yann-LeCun-puts-it-1) regarding RL being the cherry on the ML [cake.](https://miro.medium.com/max/1200/1*bvMhd_xpVxfJYoKXYp5hug.png)
+Reinforcement Learning &amp; other bedtime stories in PyTorch. This repository is [pytorch](https://pytorch.org) implementation of the [Intro RL course](https://github.com/simoninithomas/Deep_reinforcement_learning_Course) & [Advanced RL course](https://github.com/dennybritz/reinforcement-learning). Before training/playing the agent within conda env, please install the [external dependencies](https://github.com/moabitcoin/cherry-pytorch#dependencies). The name cherry is derived from Yann Lecun's [statement](https://www.quora.com/What-do-you-think-about-reinforcement-learning-Is-it-the-cherry-on-the-cake-as-Yann-LeCun-puts-it-1) regarding RL being the cherry on the ML [cake.](https://miro.medium.com/max/1200/1*bvMhd_xpVxfJYoKXYp5hug.png).
 
 ## :computer: Installation
+```
+Tested with ubuntu:18.04 only
+```
 ### :snake: Conda
 ```
+# After installed external dependencies
 conda create -n cherry python=3.6
 conda activate cherry
 pip install -r requirements.txt
 pip install -e .
 ```
 ### :whale: Docker
+```
+# CPU
+make install dockerfile=Dockerfile.cpu dockerimage=moabitcoin/cherry-pytorch-cpu
```

Comment:
Can this be simplified to

    make

since the rest is set by default already, no?

Reply:
just to easily clarify diff in cpu/gpu with input args.